system/lib/standalone/standalone.c

```diff
@@ -64,30 +64,30 @@ int clock_getres(clockid_t clk_id, struct timespec *tp) {
 // mmap support is nonexistent. TODO: emulate simple mmaps using
 // stdio + malloc, which is slow but may help some things?
 
-intptr_t _mmap_js(size_t length,
-                  int prot,
-                  int flags,
-                  int fd,
-                  size_t offset,
-                  int* allocated) {
+#if 0
```

Comment:
Is this stale?

Reply:
Ah, thanks, this is indeed left over... cleaning up.